docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -57,9 +57,15 @@ The DaemonSet integrates {elastic-endpoint} into your Kubernetes cluster. The {a
 
 You first need to download the DaemonSet manifest `.yaml`, then modify it to include your {fleet} URL and Enrollment Token before you deploy it to the clusters you want to monitor.
 
-1. Download the DaemonSet manifest using the command: `curl -L -O https://raw.githubusercontent.com/elastic/main/releases/8.4.0/kubernetes/deploy/elastic-endpoint-security.yaml`
-2. Fill in the manifest's `FLEET_URL` field with your {fleet} server's `Host URL`. To find it, go to **{kib} -> Management -> {fleet} -> Settings**. For more information, refer to {fleet-guide}/fleet-settings.html[Fleet UI settings].
-3. Fill in the manifest's `FLEET_ENROLLMENT_TOKEN` field with a Fleet enrollment token. To find one, go to **{kib} -> Management -> {fleet} -> Enrollment tokens**. For more information, refer to {fleet-guide}/fleet-enrollment-tokens.html[Fleet enrollment tokens].
+. Download the DaemonSet manifest using this command: 
++
+[source,console]
+----
+curl -L -O https://raw.githubusercontent.com/elastic/endpoint/main/releases/8.4.0/kubernetes/deploy/elastic-endpoint-security.yaml
```

Comment:
I'm wondering if the version number in this URL should be a variable (Asciidoc calls them attributes) instead of hard-coded. For example, once we publish the 8.5.0 docs, I assume we'll want to give users the 8.5.0 version of the YAML, not keep pointing back to 8.4.0.

We have a few version attributes to choose from ([here](https://github.com/elastic/docs/blob/master/shared/versions/stack/8.4.asciidoc) are the ones for 8.4), depending on whether this URL is going to change per major, minor, or patch version. Assuming it's all three, then this would probably work:
```suggestion
curl -L -O https://raw.githubusercontent.com/elastic/endpoint/main/releases/{version}/kubernetes/deploy/elastic-endpoint-security.yaml
```

Reply:
Hi @joepeeples, Two things are leaning me away from this:

1) For the 8.6/8.7 we are planning to move away from using this for pulling down the proper YAML file. What we want to do is have a unified onboarding process within Kibana, where the YAML file is generated dynamically based upon the integrations that you have. This includes agent, endpoint and even Cloudbeats (CSP). So this doc will need to be changed, and most definitely the curl console command will be removed. The Beta release for 8.4 is an interim step patched by documentation.

2) I hesitate to bind a release to file that may or may not exist within another repository because we haven't changed or populated it with a new version update. (Hopefully, we won't change this YAML file before we get 1. in place). If we release 8.5 and we do not create a new path within https://github.com/elastic/endpoint/releases/xxx , i.e. we might have forgotten about this binding, and when we release 8.5, we will be presenting to the user a file that doesn't exist.

Reply:
@qcorporation Totally makes sense, thanks for the helpful context! Then I agree hard-coding is a good choice. I guess if possible, we'll try to remember to confirm this URL in future docs releases, but as you say, we may not need to include the URL in the future anyway. cc: @benironside 

Looks like CI is happy, and so am I! 👍